presentation/src/main/java/com/madrid/presentation/viewModel/libraryViewModel/LibraryScreenState.kt

```diff
@@ -11,6 +13,9 @@ data class LibraryScreenState(
     val historyList: List<MediaUiState> = listOf(),
     val refreshState: Boolean = false,
     val isGuest: Boolean = false,
+    val showCreateListBottomSheet: Boolean = false,
```

Comment:
`showCreateListBottomSheet` this naming means it's an action not a state , you should name it to something like `isCreateListBottomSheetOpen` or `isCreateListBottomSheetShown`